srcpkgs/gdm/template

```diff
@@ -25,10 +25,10 @@ short_desc="GNOME Display Manager"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/GDM"
-changelog="https://gitlab.gnome.org/GNOME/gdm/-/raw/gnome-45/NEWS"
 #changelog="https://gitlab.gnome.org/GNOME/gdm/-/raw/main/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/gdm/-/raw/46.0/NEWS/"
```

Comment:
This should be `https://gitlab.gnome.org/GNOME/gdm/-/raw/gnome-46/NEWS` (the 46.0 is the tag for the 46.0 release and doesn't contain the changelogs for the whole release series)

And since the gnome-46 branch hasn't been created yet, you can comment it and uncomment the one above.

Reply:
fixed